database/elastic_mapping/elasticMapFactory.ts

```diff
@@ -16,6 +16,7 @@ export default {
 
     return templates.reduce(
       (baseMapping: any, template: TemplateSchema) =>
+        // eslint-disable-next-line max-statements
```

Comment:
@Zasa-san as far as possible we should try to avoid disabling ESLint issues, please can you extract some code into a function in order to pass the check instead

Reply:
I can extract lines 38:51. And replace it for 
`map = inheritedPropertyMapping(map, property, topicClassification);`
where `inheritedPropertyMapping` does what is in 38:51 right now.

Since this is all part of a reduce that appends values to `map` I would say it's ok to ignore the max statements lint issue, unless the function mapping function continues to grow.

Reply:
I agree, a refactor it's not needed, I'll merge it

Reply:
Thanks Mercy!